### DISCUSSION TITLE
Feature: Choose the starting day of the week on calendar

### DISCUSSION THREAD

**Discussion by xlucasbitencourt** [rewindui/rewindui#111] [Ideas]
I like to propose to add the prop firstDayOfTheWeek, which can change the first day the appears on the week column. Right now the week starts on Monday, and there are some location that the week starts on Sunday, so it would be a good idea to choose the right day to start.

**Comment by NickDunas** [comment]
Hello @xlucasbitencourt!

I thought of adding this option while developing the Calendar component, but wasn't really sure if anyone might actually need it. It won't be that hard to do so, so I think I will go with it once I find some time.

Thank you for reaching out!
